demo/loader/entries/boot.conf

```diff
@@ -0,0 +1,4 @@
+title       Grisp2 demo
+options     root=/dev/mmcblk1p2 rootflags='discard,data=journal'
```

Comment:
Do we really need these options?

Reply:
I don't know, they were there before in the boot entries. Now it's just options for the demo though.

Reply:
For Erlang we don't even set the `options` field at all.